river/Config.zig

```diff
@@ -76,6 +77,7 @@ modes: std.ArrayListUnmanaged(Mode),
 float_rules: RuleList(bool) = .{},
 ssd_rules: RuleList(bool) = .{},
 tag_rules: RuleList(u32) = .{},
+output_rules: RuleList(*Output) = .{},
```

Comment:
Using `*Output` here is problematic, because this way it's only possible to create rules for currently existing outputs. Also the pointer and therefore the rule will become invalid when the output disconnects (should the output reconnect, the pointer is still invalid).

It's probably better to use output name / identifier here instead.